src/components/ProgressBar/ProgressBar.meta.tsx

```diff
@@ -0,0 +1,44 @@
+import { ProgressBar, ProgressBarProps } from '.';
+import Registry from '@ui-autotools/registry';
+
+const ProgressBarMetadata = Registry.getComponentMetadata(ProgressBar);
+ProgressBarMetadata.nonReactStrictModeCompliant = true;
+
+ProgressBarMetadata.addSim({
+  title: 'render',
+  props: {
+    value: 20,
+  } as ProgressBarProps,
+});
+
+ProgressBarMetadata.addSim({
+  title: 'render with percentage',
+  props: {
+    value: 20,
+    showProgressIndication: true,
+  } as ProgressBarProps,
+});
+
+ProgressBarMetadata.addSim({
+  title: 'render with percentage 33%',
+  props: {
+    value: 33,
+    showProgressIndication: true,
+  } as ProgressBarProps,
+});
+
+ProgressBarMetadata.addSim({
+  title: 'render with percentage 100%',
+  props: {
+    value: 33,
+    showProgressIndication: true,
+  } as ProgressBarProps,
+});
+
+ProgressBarMetadata.addSim({
+  title: 'render with percentage RTL',
+  props: {
+    value: 33,
+    showProgressIndication: true,
+  } as ProgressBarProps,
+});
```

Comment:
This actually won't work
but it's ok without it also, so you can remove this one